pytensor/tensor/basic.py

```diff
@@ -2758,29 +2758,21 @@ def concatenate(tensor_list, axis=0):
     return join(axis, *tensor_list)
 
 
-def horizontal_stack(*args):
```

Comment:
Can you keep the original function names as well with a deprecation warning not to break people's code?

```python
def horizontal_stack(*args):
  warnings.warn("horizontal_stack was renamed to hstack and will be removed in a future release", FutureWarning)
  return hstack(*args)
```